tests/bookController.test.js

```diff
@@ -182,75 +223,70 @@ describe('Book Controller Tests:', () => {
   describe('Patch Book', () => {
     it('should return patched book.', async () => {
       // given
-      const bookId = '507f191e810c19729de860ea';
-      const mockBook = mockBooks[0];
-      mockBook._id = bookId;
+      const bookId = 1;
+      const mockBook = mockBooks.find((book) => book.id === bookId);
       const patchedBook = {
         read: true
       };
       const updatedBook = mockBook;
       updatedBook.read = true;
-      mockingoose(model).toReturn(updatedBook, 'save');
+      mockBook.update = () => updatedBook;
 
       // when
-      const controller = bookController(model);
+      const controller = bookController();
       const returnedBook = await controller.patch(mockBook, patchedBook);
 
       // then
-      expect(JSON.parse(JSON.stringify(returnedBook))).toMatchObject(updatedBook);
+      expect(returnedBook).toMatchObject(updatedBook);
     });
 
     it('id should not be updated.', async () => {
       // given
-      const bookId = '507f191e810c19729de860ea';
-      const mockBook = mockBooks[0];
-      mockBook._id = bookId;
+      // given
+      const bookId = 1;
+      const mockBook = mockBooks.find((book) => book.id === bookId);
```

Comment:
allg. zur Arrow Funktion in JS kannst du () weglassen, wenn es da nur ein Attribut gibt also 
`.find(book => bookId === bookId)`

Reply:
ESLint beschwert sich immer wenn ich die parens weglasse 😄 
Es sieht ja so aus, dass by default diese Regel auf "always" gesetzt wird:
https://eslint.org/docs/rules/arrow-parens
ich würde aber die Regel bearbeiten und auf "as-needed" setzen.